assignments/function-conversion.js

```diff
@@ -3,28 +3,38 @@
 // let myFunction = function () {
 // console.log("Function was invoked!");
 // };
-// myFunction();
+
+const myFunction = ()=> console.log("Function was invoked!");
+
+myFunction();
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
-// anotherFunction("Example");
+const anotherFunction = (param) => param;
```

Comment:
I believe you already know this, but in functions with just 1 param you could skip the parenthesis for added cleanness.